ui/src/features/annotator/tools/segment-anything-tool/interactive-segmentation-point.component.tsx

```diff
@@ -11,7 +11,7 @@ interface InteractiveSegmentationPointProps extends InteractiveAnnotationPoint {
 export const InteractiveSegmentationPoint = ({ x, y, positive, isLoading }: InteractiveSegmentationPointProps) => {
     const { scale } = useZoom();
     const fill = positive ? 'var(--brand-moss)' : 'var(--brand-coral-cobalt)';
-    const animationScale = `calc(1 / var(--zoom-scale))`;
+    const animationScale = 1 / scale;
     const pointRadius = 5 / scale;
```

Comment:
Could we keep using css variables here instead? That will give a much smoother zoom in / out experience.